### PR TITLE
Blog app - Add forms

### PR DESCRIPTION
**Pull request by DeVuDeveloper**
Hi,
- created forms for adding posts, comments, and likes as partial
- added logic in controllers for creating posts, likes, and comments
- added links in views files
-creating likes, comments and posts based on current_user
-optional: added flash notices and alerts
- when create posts, likes and comments they are saved in database
- all counters works properly
Update:
REquested changes made: added methods new in all controllers

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
   end
 
   def show
     @user = User.find(params[:user_id])
     @post = @user.posts.find(params[:id])
   end
+
+  def new
+    @current_user = current_user
+  end
```
Comment (mikethreels) [inline app/controllers/posts_controller.rb]: - [ ] the `new` method in the `PostController` should look something like below:
```
    def new
      @post = Post.new
    end

```

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
```
Comment (mikethreels) [inline app/controllers/comments_controller.rb:1]: - [ ] there should also be a `new` method in the `CommentsController`

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Dejan, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Amrhub — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi @VuDej , 👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights ✅ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**